tests/PropertyClass.php

```diff
@@ -0,0 +1,18 @@
+<?php
+class PropertyClass {
```

Comment:
place braces according to [PSR-2](https://www.php-fig.org/psr/psr-2/#1-overview)

```
class A
{
  function method($args)
  {
  }
}
```